src/EditorFeatures/CSharpTest/CodeActions/ConvertIfToSwitch/ConvertIfToSwitchTests.cs

```diff
@@ -691,6 +691,7 @@ void M(object o)
         {
             case string s when s.Length > 5 && s.Length < 10:
                 M(o: 0);
+
```

Comment:
note: i woudl likely only preserve trivia if there's something non-whitespace in it.

Reply:
I'm somewhat on the fence, maybe leaning the other way. Can we do the most simple thing possible until someone requests a change?